nservicebus/scheduling/index.md

```diff
@@ -18,6 +18,7 @@ Scheduling a task depends on [delayed delivery](/nservicebus/messaging/delayed-d
 The Scheduler leverages the queuing system to trigger scheduled actions. Under heavy load there may be some disparity between the expected time of a scheduled action and execution time due to the delay between timeout messages being generated and processed. On approach to mitigating this behavior is to run the Scheduler in a dedicated endpoint so that appropriate resource allocation can put in place to ensure timely execution of scheduled actions. Or, alternatively, use a difference scheduling technology, for example:
 
  * A [.NET Timer](https://msdn.microsoft.com/en-us/library/system.threading.timer.aspx).
+ * [NServiceBus Sagas](/nservicebus/sagas/)
```

Comment:
We do mention this below as well but I think it should be here as well to make the list of options complete